fontdrasil/src/lib.rs

```diff
@@ -0,0 +1 @@
+pub mod orchestration;
```

Comment:
root `lib` should have `//! at least some module comment`. This is the text on the root docs page for this crate.

Also, I think it's worth turning on `deny(missing_docs)`; it can be annoying, but it has real benefits, and it's even more annoying to turn on in three months and then have 300 errors to fix. You can always judiciously use `#[allow(missing_docs)]` on specific items where you don't think docs are useful, like large enums.

Reply:
Added a string here and filed https://github.com/googlefonts/fontmake-rs/issues/85 to make sure we turn this on globally. It's a great idea but I don't want to fill up this PR with doc additions.

Reply:
oh yes definitely not for this PR :)